src/render/solver.rs

```diff
@@ -383,3 +383,162 @@ pub async fn install_packages(
 
     Ok(())
 }
+
+/// Extracts short channel name from URL (e.g., "conda.anaconda.org/conda-forge" → "conda-forge").
+fn short_channel(channel: Option<&str>) -> String {
+    let channel = channel.unwrap_or_default();
+    if channel.contains('/') {
+        channel
+            .rsplit('/')
+            .find(|s| !s.is_empty())
+            .unwrap_or_default()
+            .to_string()
+    } else {
+        channel.to_string()
+    }
+}
+
+/// Creates a formatted table for externally managed environment info.
+pub fn create_environment_table(name: &str, required_packages: &[RepoDataRecord]) -> String {
```

Comment:
I might be missing something, but this function is not used except for the tests?